pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -58862,7 +58862,6 @@ self: {
        license = lib.licenses.bsd3;
        hydraPlatforms = lib.platforms.none;
        mainProgram = "cabal-add";
-       broken = true;
```

Comment:
`./maintainers/scripts/haskell/regenerate-hackage-packages.sh` is currently broken so I removed by hand